src/Illuminate/Database/Eloquent/Model.php

```diff
@@ -546,6 +546,22 @@ public function fill(array $attributes)
         return $this;
     }
 
+    /**
+     * Fill only the missing with an array attributes of the model.
+     *
+     * @param  array  $attributes
+     * @return $this
+     */
+    public function fillMissing(array $attributes)
+    {
+        $attributes = collect($attributes)
+            ->filter(fn ($value, $key) => Arr::has($this->attributesToArray(), $key))
+            ->reject(fn ($value, $key) => filled($this->$key))
```

Comment:
Helper functions inside the Database namespace is not good

Reply:
@jmarcher What would be a better solution? 
This is also used in the `replicate` function, should it be changed there too?